style/index.less

```diff
@@ -0,0 +1,18 @@
+.content {
+  margin: 20px 10%;
+
+  [data-wikipedia-preview] {
+    background-color: #eaf3ff;
+    padding: 2px 2px 2px 2px;
```

Comment:
This padding (and the :after icon) will make the whole text jump since this is applied after the initial page rendering.

Reply:
Yea the padding and icon are some of the CSS specifics that Sudhanshu and I are still refining. What do you mean by "make the whole text jump" exactly?

Reply:
The content of the partner site will be first rendered, then this library will be loaded at the end of the body and this style will be applied. When it does, all those links will become 4px wider, which will make most of the text move. 

Similar to when you have an image that loads late without an appropriate placeholder: everything below the image jump down.

Reply:
Gotcha. I'm researching more about what we discussed earlier this week, see if it's possible to load the style in the header first and then the script at the end of the body. 

From what I can gather after some initial investigation, according to this `style-loader` docs [1] the default behavior is that it inserts the CSS in the `<head>` tag already. Inspecting the HTML source locally I do see `style` tags with the CSS, but I need to look into this further to actually confirm/verify the load order. 

[1] - https://github.com/webpack-contrib/style-loader#insert